components/03-modules/footer/footer.config.js

```diff
@@ -0,0 +1,170 @@
+module.exports = {
+  context: {
+    linkList: {
+      class: 'dropdown-list@screen-m js-is-open-screen-m dropdown-list--dark',
```

Comment:
I see that you use a bemit namespacing convention in a first class, but the second one has no `@`. Why ?

Reply:
it's class created for JS and used only in JS. It's b/c can't use '@' i.e. for querySelector - it's returned an error. `Failed to execute 'querySelector' on 'Document': '.js-is-open@screen-m' is not a valid selector.`

Reply:
Did you try using escape chars ?

Reply:
you are right. Class changed to `dropdown-list--is-open@screen-m`